src/api/upload.ts

```diff
@@ -0,0 +1,46 @@
+import { RouteInfo, UploadFile, UploadFileMetadata } from '~/types'
+import { uploadFilesToUrls } from './athena'
+import { getAlreadyUploadedFiles, requestToUploadFiles } from './file'
+import { parseRouteName } from './route'
+
+export const FileTypes = {
+  logs: ['rlog.bz2', 'rlog.zst'],
+  cameras: ['fcamera.hevc'],
+  dcameras: ['dcamera.hevc'],
+  ecameras: ['ecamera.hevc'],
+}
+
+const getFiles = async (routeName: string, types?: (keyof typeof FileTypes)[]) => {
```

Comment:
Is it worth adding this?

```suggestion
type FileType = keyof typeof FileTypes

const getFiles = async (routeName: string, types?: FileType[]) => {
```